### PR TITLE
Add highlight for search

### PR DESCRIPTION
**Pull request by IsaMic**

### CODE REVIEW

File: Helper/Search/QueryBuilder.php [hunk 1]
```diff
@@ -28,6 +28,7 @@ public function buildBody(): array
             'aggs' => $this->getAggs($hasPostFilter),
             'suggest' => $this->getSuggest(),
             'sort' => $this->getSort(),
+            'highlight' => $this->getHighlight()
```
Comment (dameert) [inline Helper/Search/QueryBuilder.php]: I think we can use $this->search->getHighlight() directly ?
The return null of getHighlight is not necessary, as array_filter considers [] as a value to remove.
@davma what is your opinion?

File: Helper/Search/Search.php [hunk 2]
```diff
@@ -207,6 +210,16 @@ public function getSorts(): array
     {
         return $this->sorts;
     }
+    
+    public function hasHighlight(): bool
```
Comment (dameert) [inline Helper/Search/Search.php]: Not needed if my previous remark is taken into account.

File: Helper/Search/QueryBuilder.php [hunk 3]
```diff
@@ -172,4 +173,13 @@ private function getSort(): ?array
             ]
         ];
     }
+    
+    private function getHighlight(): ?array
```
Comment (dameert) [inline Helper/Search/QueryBuilder.php]: Function seems to be not needed.

File: Helper/Search/QueryBuilder.php (hunk 1 above)
Comment (Davidmattei) [reply Helper/Search/QueryBuilder.php]: Yes that would indeed be beter!

**Pull request merged**